app/uk/gov/hmrc/tariffclassificationfrontend/views/forms/components/input_textarea.scala.html

```diff
@@ -30,23 +30,26 @@
 <div class="form-group @if(field.hasErrors){form-group-error}">
     <label class="form-label" for="@{field.id}">
         <span class="@labelClass">@label</span>
-        @hint.map { hint =>
-        <span class="form-hint">@hint</span>
-        }
-        @field.errors.map { error =>
-        <span class="error-message" id="error-message-@{field.id}-input">@customErrorMessage.getOrElse( messages(error.message, error.args: _*))</span>
-        }
     </label>
 
+    @hint.map { hint =>
+        <span class="form-hint" id="@{field.id}-hint">@hint</span>
+    }
+
+    @field.errors.map { error =>
+        <span class="error-message" id="error-message-@{field.id}-input">@customErrorMessage.getOrElse( messages(error.message, error.args: _*))</span>
+    }
+
     <textarea
             class="form-control w-100 @inputClass"
             id="@{field.id}"
             name="@{field.id}"
-            aria-describedby="error-message-@{field.id}-input"
+            aria-describedby="@{field.id}-hint"
```

Comment:
Looking at the guidelines, this should be something like
aria-describedby="@{field.id}-hint error-message-@{field.id}-input" in case of error being present (so referencing both normal hint and error).